revolution_pi/gpio_chip.go

```diff
@@ -67,6 +69,10 @@ func (g *gpioChip) showDeviceList() error {
 	for i := 0; i < int(cnt); i++ {
 		if deviceInfoList[i].i8uActive != 0 {
 			g.logger.Debugf("device %d is of type %s is active", i, getModuleName(deviceInfoList[i].i16uModuleType))
+			if deviceInfoList[i].isDIO() {
+				g.logger.Info("device info: ", deviceInfoList[i])
+				g.dioDevices = append(g.dioDevices, deviceInfoList[i])
```

Comment:
during setup, I grab each DIO related device to use with pin configuration. This lets me differentiate between multiple DIOs